File: src/001_sum/sum.java
class Main {

    private static final int N = 500000;

    private static void bench() {
        long result = 0;
        for (int i = 0; i < N; i++) {
            result += i;
        }

        System.out.println("loops=" + N);
        System.out.println("result=" + result);
    }

    public static void main(String args[]) {
        long t1 = System.currentTimeMillis();
        bench();
        long cost_time = System.currentTimeMillis() - t1;
        System.out.println("cost_time:" + cost_time + "ms");
    }
}

File: src/001_sum/sum.py
# -*- coding:utf-8 -*-
# @author xupingmao
# @since 2022/01/26 10:25:00
# @modified 2022/01/26 10:32:43
# @filename sum.py
import sys
import time

def timeit(func, *args):
    t1 = time.time()
    ret = func(*args)
    t2 = time.time()
    cost_time = (t2-t1) * 1000
    print("cost_time:%sms" % cost_time)

def main():
    N = 500000
    result = 0
    for i in range(N):
        result += i

    print("loops=%d" % N)
    print("result=%d" % result)

if __name__ == '__main__':
    timeit(main)